fenix/app/src/test/java/org/mozilla/fenix/settings/SettingsFragmentTest.kt

```diff
@@ -98,6 +99,7 @@ class SettingsFragmentTest {
     }
 
     @Test
+    @org.robolectric.annotation.Config(sdk = [Build.VERSION_CODES.Q])
```

Comment:
Could we have a test to verify that the setting is hidden when we have a version code for Android 9 or something?

Reply:
Added :)